.github/workflows/push-main.yml

```diff
@@ -83,11 +84,11 @@ jobs:
 
       - name: Configuring container registry credentials
         run: |
-          echo "${{ steps.auth.outputs.access_token }}" | skopeo login -u oauth2accesstoken --password-stdin https://europe-docker.pkg.dev
+          echo "${{ steps.auth.outputs.access_token }}" | skopeo login -u oauth2accesstoken --password-stdin https://${{env.IMAGE_REPO}}
```

Comment:
@aledbf can you think of another way to login with skopeo w/o echoing the access token? I know it's only valid for an hour, I'm just trying to avoid it being potentially output to logs and used.